code/modules/uplink/uplink_items.dm

```diff
@@ -1585,7 +1594,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 /datum/uplink_item/race_restricted/syndilamp
 	name = "Extra-Bright Lantern"
 	desc = "We heard that moths such as yourself really like lamps, so we decided to grant you early access to a prototype \
-	Syndicate brand \"Extra-Bright Lantern™\". Enjoy."
+	Syndicate brand \"Extra-Bright Lantern�\". Enjoy."
```

Comment:
wut

Reply:
I have no idea I checked the code before commititng and it did say ™. I check after and it still was like this

Reply:
Fixed